account_journal_lock_date/models/account_move.py

```diff
@@ -12,11 +12,6 @@ class AccountMove(models.Model):
 
     _inherit = "account.move"
 
-    def write(self, values):
-        res = super().write(values)
-        self._check_fiscalyear_lock_date()
```

Comment:
Not which code is going to call the `_check_fiscalyear_lock_date` method if you remove this call ?

Maybe we have to check according values write on the account move but I think move should be reconciled before lock the fiscal year

Reply:
The original method _check_fiscalyear_lock_date is defined here https://github.com/odoo/odoo/blob/70a0ad280e0c8a62f24d6943be082298fa98e443/addons/account/models/account_move.py#L1831 and it's already call many time in this same file : 

- https://github.com/odoo/odoo/blob/70a0ad280e0c8a62f24d6943be082298fa98e443/addons/account/models/account_move.py#L2012
- https://github.com/odoo/odoo/blob/70a0ad280e0c8a62f24d6943be082298fa98e443/addons/account/models/account_move.py#L2017
- https://github.com/odoo/odoo/blob/70a0ad280e0c8a62f24d6943be082298fa98e443/addons/account/models/account_move.py#L4097
- ...

For me, we only need to override this method, not add a new call.

Reply:
ok I see, at least you'll have to update failing unit test according that change, I'm wondering we should probably keep a way to prevent journal entry name to be updated once period is locked (the failing test)